marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/endpoints/collector.sjs

```diff
@@ -41,13 +41,6 @@ if(method === 'GET') {
   const database = requestParams.database;
   let options = requestParams.options ? JSON.parse(requestParams.options) : {};
 
-  let jobDoc = datahub.jobs.getJobDocWithId(jobId);
```

Comment:
we do not need to have the lines:
https://github.com/marklogic/marklogic-data-hub/blob/29f69f0379c4a7167e69baa5178073f22d46bb02/marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/endpoints/collector.sjs#L40-L41